feature-libs/product/configurators/rulebased/index.ts

```diff
@@ -1,6 +1,9 @@
 export * from './components/index';
 export * from './core/index';
-export * from './occ/variant/index';
+export { CpqConfiguratorRestModule as ɵCpqConfiguratorRestModule } from './cpq/cpq-configurator-rest.module';
```

Comment:
Remove this line, should be done in cpq/index.ts, and we should expose this module